README.md

```diff
@@ -1,8 +1,207 @@
-
 # goods-movement-system-port-api
+This API provides resources from the Goods Vehicle Movement Service (GVMS) for ports and border locations.
+
+GVMS is used by hauliers to declare all goods movements crossing certain UK roll-on-roll-off ports.
+
+This API is used by ports and border locations to:
+
+* Get a list of controlled GMR details that have embarked to or arrived at a GVMS port or border location
+* Get a list of controlled GMR details that are checked-in for departure at a GVMS port or border location
+* Get a list of checked-in GMR details at a GVMS port or border location
+* Get GVMS reference data
+
+## Running the service locally
+By default, the service runs on 8998 port.
+
+#### Using service manager 2
+
+The following sm2 command starts the service but its also part of `GMS_ALL` profile
+
+`sm2 --start GOODS_MOVEMENT_SYSTEM_PORT_API`
+
+the logs can be viewed by running `sm2 --logs GOODS_MOVEMENT_SYSTEM_PORT_API` in a different terminal window
+
+#### Using sbt
+For local development use `sbt run` but if its already running in sm2, we should stop it first before running sbt command
+
+The endpoint in this API is application-restricted.
+
+All ports requiring access to this API must be pre-authorized by HMRC.
+
+service logs can be viewed by running `sm --logs GOODS_MOVEMENT_SYSTEM_PORT_API`
+
+```GET  /:portId/arrivals/controlled```
+
+Port can pull all inspection required GMR's that are arriving at their port.
```

Comment:
If possible, can we change this: 

`Port can pull all inspection required GMR's that are arriving at their port.`

to this: 

Retrieve all GMRs arriving at a specified port which require an inspection."